### PR TITLE
feat(connectors): inbound deduplication

### PR DESCRIPTION
**Pull request by chillleader**
## Description

Documentation for the new inbound connector deduplication mechanism. This functinality is related to all inbound connectors, so I decided to place the documentation on the page `Using inbound connectors`.

I've also introduced tabs to this page to hide the repeating information in the beginning for different element types.

Note: these changes will be introduced with 8.6-alpha1.

closes https://github.com/camunda/team-connectors/issues/672

related epic: https://github.com/camunda/product-hub/issues/1996

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] ~~I have added changes to the relevant `/versioned_docs` directory~~, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or ~~they are not for **future versions**.~~
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/connectors/use-connectors/inbound.md [hunk 1]
```diff
@@ -87,9 +119,65 @@ For example, given that your correlation key is defined with `requestIdValue` pr
...
+
+If the automatic deduplication rules do not work for your use case, you can manually assign a deduplication ID to each Connector event.
+This allows you to group Connectors in a more flexible way, based on your requirements.
+
+If needed, you can have multiple Connectors with the same properties that have different deduplication IDs. This way, you can still have multiple instances of the same Connector listening to the same event source, but each instance will have its own deduplication ID and will be treated as a separate entity by the Connector runtime.
+
+To assign a deduplication ID, you need to do the following:
+
+1. Enable the **Manual mode** checkbox in the **Deduplication** section of the Connector properties.
+2. The **Deduplication ID** field will appear. Fill in the desired deduplication ID.
```
Comment (johnBgood) [inline docs/components/connectors/use-connectors/inbound.md:161]: It might be obvious, but it would be nice to have an example here

Comment (chillleader) [reply docs/components/connectors/use-connectors/inbound.md:161]: I added a screenshot with example input below

**Review by johnBgood — APPROVED** [review]
Looks clear to me 👌

File: docs/components/connectors/use-connectors/inbound.md [hunk 2]
```diff
@@ -87,9 +119,67 @@ For example, given that your correlation key is defined with `requestIdValue` pr
...
+To avoid this, both events can be assigned to the same subscription by assigning the same **Deduplication ID** to both events. Then, all messages will be consumed by the same subscription, and the Connector runtime will evaluate the **Activation condition** of each event to determine which one should be triggered.
+
+:::note
+When using this pattern, make sure that the **Activation condition** of each event is mutually exclusive, so that only one event is triggered for each message.
+Attempting to trigger multiple events for the same message will result in an error.
+:::
+
+### Automatic deduplication
+
+By default, the Connector runtime will assign the same deduplication ID to Connector events that have equal properties, and different deduplication IDs to events that have different properties.
```
Comment (sbuettner) [inline docs/components/connectors/use-connectors/inbound.md:145]: We might also want to mention that equality exactly the same values including whitespaces.

**Review by sbuettner — APPROVED** [review]

**Review by YanaSegal — DISMISSED** [review]
Well explained! LGTM

**Review by christinaausley — APPROVED** [review]
Nice writing!

However, I will note that anchor links are not supported for tabbed sections of docs. See [here](https://github.com/camunda/camunda-docs/pull/3445) -- just an FYI in the case we are trying to link to this content elsewhere.

**Comment by chillleader** [comment]
Thanks for the review and for the heads-up @christinaausley, I think we don't link this content anywhere at the moment, but good to know!

I just noticed one more minor thing that seemed off grammatically, please take another look 🙂

**Review by christinaausley — APPROVED** [review]

**Pull request merged**